core/cap-0033.md

```diff
@@ -275,11 +290,12 @@ enum UpdateSponsorshipResultCode
     UPDATE_SPONSORSHIP_SUCCESS = 0,
 
     // codes considered as "failure" for the operation
-    UPDATE_SPONSORSHIP_DOES_NOT_EXIST = -1,
-    UPDATE_SPONSORSHIP_NOT_SPONSOR = -2,
-    UPDATE_SPONSORSHIP_LINE_FULL = -3,
-    UPDATE_SPONSORSHIP_LOW_RESERVE = -4,
-    UPDATE_SPONSORSHIP_ONLY_TRANSFERABLE = -5
+    UPDATE_SPONSORSHIP_DOES_NOT_EXIST = -1,   // specified entry does not exist
+    UPDATE_SPONSORSHIP_NOT_SPONSOR = -2,      // not sponsor of specified entry
+    UPDATE_SPONSORSHIP_LOW_RESERVE = -3,      // owner cannot afford this entry
+    UPDATE_SPONSORSHIP_ONLY_TRANSFERABLE = -4 // sponsorship of ClaimableBalance
+                                              // must be transferred to another
+                                              // account
```

Comment:
Does this apply to only claimable balance? I would think this could happen for any sponsorship?

Reply:
Right now this can only happen for `ClaimableBalanceEntry`. Basically, every type of entry other than claimable balance has a default sponsor (the account that controls that entry, which is the account that reflects that entry in the `numSubEntries`). But claimable balance has no default, it always has to have an explicit sponsor.